src/QuizAtom.tsx

```diff
@@ -1,7 +1,322 @@
+    if (!isAnswered) {
+        return AnswerState.Unanswered;
+    } else if (isCorrect && isChosen) {
+        return AnswerState.CorrectChosen;
+    } else if (isCorrect && !isChosen) {
+        return AnswerState.CorrectUnchosen;
+    } else if (!isCorrect && isChosen) {
+        return AnswerState.IncorrectChosen;
+    } else {
+        return AnswerState.IncorrectUnchosen;
+    }
+};
+
+const neutralBackgroundColour = neutral[97];
+const darkerNeutralBackgroundColour = neutral[86];
+const correctBackgroundColour = success[400];
+const correctUnchosenBackgroundColour = 'rgb(34, 135, 77, 0.6)';
+const incorrectBackgroundColour = error[400];
+
+const commonLabelStyle = css`
+    svg {
+        width: 30px;
+        height: 30px;
+    }
+    display: block;
+    padding: 0.75rem 1.25rem;
+    transition: background-colour 100ms;
+`;
+
+const unansweredLabel = css`
+    cursor: pointer;
+    background-color: ${neutralBackgroundColour};
+    ${commonLabelStyle}
+    background-color: ${neutralBackgroundColour};
+    :hover {
+        background-color: ${darkerNeutralBackgroundColour};
+    }
+`;
+
+const correctChosenLabel = css`
+    background-color: ${correctBackgroundColour};
+    ${commonLabelStyle}
+`;
+
+const correctUnchosenLabel = css`
+    background-color: ${correctUnchosenBackgroundColour};
+    ${commonLabelStyle}
+`;
+
+const incorrectChosenLabel = css`
+    background-color: ${incorrectBackgroundColour};
+    ${commonLabelStyle}
+`;
+
+const incorrectUnchosenLabel = css`
+    background-color: ${neutralBackgroundColour};
+    ${commonLabelStyle}
+`;
+
+const labelStyle = (state: AnswerState) => {
+    switch (state) {
+        case AnswerState.Unanswered:
+            return unansweredLabel;
+        case AnswerState.CorrectChosen:
+            return correctChosenLabel;
+        case AnswerState.CorrectUnchosen:
+            return correctUnchosenLabel;
+        case AnswerState.IncorrectChosen:
+            return incorrectChosenLabel;
+        case AnswerState.IncorrectUnchosen:
+            return incorrectUnchosenLabel;
+        default:
+            return commonLabelStyle;
+    }
+};
+
+const commonTextStyle = css`
+    padding-left: 8px;
+    padding-top: 0;
+    min-height: auto;
+`;
+
+const unchosenText = css`
+    color: ${neutral[7]};
+    ${commonTextStyle};
+`;
+
+const chosenText = css`
+    color: ${neutral[100]};
+    ${commonTextStyle};
+`;
+
+const textStyle = (state: AnswerState) => {
+    switch (state) {
+        case AnswerState.IncorrectChosen:
+            return chosenText;
+        case AnswerState.CorrectChosen:
+            return chosenText;
+        default:
+            return unchosenText;
+    }
+};
+
+const revealTextStyle = (state: AnswerState) => css`
+    ${textStyle(state)}
+    ${textSans.xsmall()}
+`;
+
+const fieldsetStyle = css`
+    margin-bottom: 12px;
+    border: 0px;
+    padding: 0px;
+`;
+
+// Will appear on screen readers
+const visuallyHidden = css`
```

Comment:
Source provides this as a helper `import { visuallyHidden } from "@guardian/src-foundations/accessibility"`